### ISSUE TITLE
No POD list

### ISSUE THREAD

**Issue by GreenBombel** [kontena/lens#6647]
**Describe the bug**
There is no PODs optyion iw Workloads :)

**To Reproduce**
Open app and connect to cluster

**Expected behavior**
PODs option should be present in Workloads or anywhere else.

**Screenshots**
<img width="1454" alt="image" src="https://user-images.githubusercontent.com/98710253/203929746-167fce6e-4459-4181-811d-bf8d2747e84b.png">

**Environment (please complete the following information):**
- Lens Version: 2022.11.241650-latest
- OS: MacOS 13.0.1
- Installation method: Official site dmg

**Comment by BioQwer** [comment]
same problem
it's main window for me
```
Lens: 2022.11.241650-latest
Extension API: 6.2.2
Electron: 19.1.7
Chrome: 102.0.5005.167
Node: 16.14.2
© 2022 Mirantis, Inc.
```
<img width="139" alt="image" src="https://user-images.githubusercontent.com/5636129/203932912-d6bb94e6-b595-4c84-9cb1-8b80dd048890.png">

**Comment by ahkui** [comment]
duplicate #6645

**Comment by alexsuninsth** [comment]
I don't even have that much
<img width="219" alt="image" src="https://user-images.githubusercontent.com/108270654/203963174-5633e8ff-cdb6-4174-b87f-df7d8f115874.png">

**Comment by alexsuninsth** [comment]
MacOS 13.0.1 (Apple Silicon)
Lens: 2022.11.241650-latest
Extension API: 6.2.2
Electron: 19.1.7
Chrome: 102.0.5005.167
Node: 16.14.2

**Comment by shaifymehta** [comment]
I am also facing the same issue with Latest version

**Comment by markhc** [comment]
Same problem here @alexsuninsth 

![image](https://user-images.githubusercontent.com/11592227/203979794-f1326ad1-8a7e-40e7-ab32-a5cf394dd6a7.png)

After update I can't see any information about any of my clusters. `kubectl` command line still works fine.

**Comment by felipesanti** [comment]
Same issue

<img width="243" alt="image" src="https://user-images.githubusercontent.com/9478951/204012147-4f3b1f82-4741-4252-ba2f-0b3c4790eee4.png">


![image](https://user-images.githubusercontent.com/9478951/204012215-d7b9b797-4575-455a-9b61-a9563c8b344a.png)


Lens: 2022.11.241650-latest
Extension API: 6.2.2
Electron: 19.1.7
Chrome: 102.0.5005.167
Node: 16.14.2
© 2022 Mirantis, Inc.

**Comment by alexsuninsth** [comment]
would be nice if we could get access to an older stable version so we can continue using IDE whilst the fix is implemented

**Comment by GreenBombel** [comment]
Fixed in version `v2022.11.251411-latest`
